Streetcode/Streetcode.WebApi/Hangfire/HangfireDashboardAuthorizationFilter.cs

```diff
@@ -0,0 +1,16 @@
+using Hangfire.Dashboard;
+using Streetcode.DAL.Enums;
+
+namespace Streetcode.BLL.Services.Hangfire;
+
+public class HangfireDashboardAuthorizationFilter : IDashboardAuthorizationFilter
+{
+    public bool Authorize(DashboardContext context)
+    {
+        var user = context.GetHttpContext().User;
+
+        var isAdministrator = user.IsInRole(UserRole.MainAdministrator.ToString());
+
+        return isAdministrator;
+    }
```

Comment:
Have this been tested? Does this work correctly?
Can we validate the JWT token here or just decode and validate the user info?

Reply:
It will automatically validate JWT in AUTHORIZATION header and get a role from it. It works.